src/Commands/Authentication/LoginCommand.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Graph.Cli.Authentication;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+
+namespace Microsoft.Graph.Cli.Commands.Authentication;
+
+class LoginCommand
+{
+    private IAuthenticationService authenticationService;
+
+    public LoginCommand(IAuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    public Command Build() {
+        var loginCommand = new Command("login", "Login and store the session for use in subsequent commands");
+        var scopes = new Option<string>("--scopes", "The login scopes e.g. User.Read");
```

Comment:
the authentication method (the strategy) probably needs to be exposed here as well.